packages/foundation/src/atoms/roles/background.css

```diff
@@ -1,70 +1,70 @@
-html * {
+.o-ui-theme {
     /* LIGHT */
-    --o-ui-background-light: var(--o-ui-white);
-    --o-ui-background-1-light: var(--o-ui-white);
-    --o-ui-background-2-light: var(--o-ui-cloud-100);
-    --o-ui-background-2-hover-light: var(--o-ui-cloud-200);
-    --o-ui-background-2-active-light: var(--o-ui-cloud-500);
-    --o-ui-background-3-light: var(--o-ui-cloud-50);
-    --o-ui-background-4-light: var(--o-ui-neutral-100);
-    --o-ui-background-4-hover-light: var(--o-ui-neutral-300);
-    --o-ui-background-4-active-light: var(--o-ui-neutral-600);
-    --o-ui-background-5-light: var(--o-ui-marine-500);
-    --o-ui-background-5-hover-light: var(--o-ui-marine-700);
-    --o-ui-background-5-active-light: var(--o-ui-marine-900);
-    --o-ui-background-primary-1-light: var(--o-ui-primary-500);
-    --o-ui-background-primary-1-hover-light: var(--o-ui-primary-700);
-    --o-ui-background-primary-1-active-light: var(--o-ui-primary-900);
-    --o-ui-background-primary-1-focus-light: var(--o-ui-primary-50);
-    --o-ui-background-primary-2-light: var(--o-ui-primary-50);
-    --o-ui-background-danger-1-light: var(--o-ui-alert-500);
-    --o-ui-background-danger-1-hover-light: var(--o-ui-alert-700);
-    --o-ui-background-danger-1-active-light: var(--o-ui-alert-900);
-    --o-ui-background-danger-2-light: var(--o-ui-alert-50);
-    --o-ui-background-warning-1-light: var(--o-ui-sunray-50);
-    --o-ui-background-warning-1-hover-light: var(--o-ui-sunray-700);
-    --o-ui-background-warning-1-active-light: var(--o-ui-sunray-900);
-    --o-ui-background-warning-2-light: var(--o-ui-sunray-50);
-    --o-ui-background-positive-1-light: var(--o-ui-green-500);
-    --o-ui-background-positive-1-hover-light: var(--o-ui-green-700);
-    --o-ui-background-positive-1-active-light: var(--o-ui-green-900);
-    --o-ui-background-positive-2-light: var(--o-ui-green-50);
-    --o-ui-background-info-1-light: var(--o-ui-marine-50);
-    --o-ui-background-info-1-hover-light: var(--o-ui-marine-100);
-    --o-ui-background-info-active-light: var(--o-ui-marine-300);
-    --o-ui-background-input-selection-light: var(--o-ui-primary-100);
+    --o-ui-alias-background-light: var(--o-ui-global-white);
```

Comment:
I would not have expected to find "alias" in foundation and furthermore "alias" that are not related to a theme or a color scheme.

Reply:
![image](https://user-images.githubusercontent.com/361632/111346994-2968ac80-8655-11eb-9c32-7b3850fdb09e.png)

I am not sure if this will help explain the light/dark naming of aliases.